lib/commands/serve.js

```diff
@@ -72,7 +72,7 @@ module.exports = Command.extend({
           project: this.project
         });
 
-        return win.checkWindowsElevation(this.ui).then(function() {
+        return Win.checkIfSymlinksNeedToBeEnabled(this.ui).then(function() {
```

Comment:
@rwjblue there are 2 methods, one that is static and one that is on the prototype. The static one sets up the state, that the tests prefer to setup themselves.